src/apigateway/templates/apigTemplates.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export class ApigTemplates {
+    public static readonly REMOTE_INVOKE_TEMPLATE = `
+    <h1>
+        Invoke function on <%= ApiName %>
```

Comment:
Could we present the region/arn details in here as well?

Reply:
added arn since region is in there